src/bigquery_query.ts

```diff
@@ -246,6 +246,12 @@ export default class BigQueryQuery {
             let to = BigQueryQuery.formatDateToString(this.templateSrv.timeRange.to._d);
             query += " AND  _TABLE_SUFFIX BETWEEN \'" + from +  "\' AND \'" + to + "\' ";
         }
+        if (this.target.partitioned) {
+            console.log(BigQueryQuery.formatDateToString(this.templateSrv.timeRange.to._d,'-',  true))
+            query += " AND _PARTITIONTIME >= \'" +
```

Comment:
Use string literals for templating.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
@liorama thanks! still working on my typescript skills. Will add more type checks as I go